packages/delivery-electron/test/delivery.test-main.ts

```diff
@@ -207,6 +207,39 @@ describe('delivery: electron', () => {
     })
   })
 
+  it('does not attempt to re-send oversized payloads', done => {
```

Comment:
this test fails because the delivery is depending on an old version. Am I missing something here? Do we need to make and merge the changes to core separately, then bump the version of core used by various components such as delivery-electron?